src/client/refresh/vk/vk_buffer.c

```diff
@@ -38,33 +38,27 @@ static void copyBuffer(const VkBuffer *src, VkBuffer *dst, VkDeviceSize size)
 }
 
 // internal helper
-static void createStagedBuffer(const void *data, VkDeviceSize size, qvkbuffer_t *dstBuffer, qvkbufferopts_t bufferOpts, qvkbuffer_t *stagingBuffer)
+static void createStagedBuffer(const void *data, VkDeviceSize size, qvkbuffer_t *dstBuffer, qvkbufferopts_t bufferOpts)
```

Comment:
You've mixed formatting changes with functional changes. This makes it extremely hard to check what was actually changed.

Reply:
thank you, all codestyle changes moved to 695a9e9